test/parseDetectorTables.test.ts

```diff
@@ -0,0 +1,26 @@
}, { columnName: "Message", dataType: "String", columnType: null }, { columnName: "Data.Name", dataType: "String", columnType: null }, { columnName: "Data.Value", dataType: "String", columnType: null }, { columnName: "Expanded", dataType: "String", columnType: null }, { columnName: "Solutions", dataType: "String", columnType: null }], rows: [["Critical", "Your app failed immediately", "Observation", "Your app failed to start almost immediately.", "False", "null"], ["Critical", "Your app failed immediately", "Possible reason", "Your Docker image may not exist.", "False", "null"], ["Critical", "Your app failed immediately", "Error message", "docker container could not be started: naturins-linux_1", "False", "null"], ["Critical", "Your app failed immediately", "Suggestion", "Please check the repository name, image name, and container definitions.", "False", "null"]] }, renderingProperties: { type: 7, title: null, description: null } }, { table: { tableName: "", columns: [{ columnName: "Status", dataType: "String", columnType: null }, { columnName: "Message", dataType: "String", columnType: null }, { columnName: "Data.Name", dataType: "String", columnType: null }, { columnName: "Data.Value", dataType: "String", columnType: null }, { columnName: "Expanded", dataType: "String", columnType: null }, { columnName: "Solutions", dataType: "String", columnType: null }], rows: [["Info", "Get full Docker logs", "Full Docker logs can be found from: ", "<a href=\"https://naturins-linux.scm.azurewebsites.net/api/logs/docker\" target=\"_blank\">Get JSON with Docker log links</a> ", "False", "null"]] }, renderingProperties: { type: 7, title: null, description: null } }, { table: { tableName: "", columns: [{ columnName: "Time", dataType: "DateTime", columnType: null }, { columnName: "Instance", dataType: "String", columnType: null }, { columnName: "Facility", dataType: "String", columnType: null }, { columnName: "FailureCount", dataType: "Int64", columnType: null }], rows: [["2019-07-26T20:40:00", "467983_Small_90", "naturins-linux", 1]] }, renderingProperties: { defaultValue: 0, graphType: 1, graphOptions: null, timestampColumnName: null, roleInstanceColumnName: null, seriesColumns: null, type: 2, title: "Number of app Start Failures", description: null } }, { table: { tableName: "", columns: [{ columnName: "Time", dataType: "DateTime", columnType: null }, { columnName: "Instance", dataType: "String", columnType: null }, { columnName: "Facility", dataType: "String", columnType: null }, { columnName: "Message", dataType: "String", columnType: null }, { columnName: "FailureCount", dataType: "Int64", columnType: null }], rows: [["2019-07-26T20:40:00", "467983_Small_90", "naturins-linux", "docker container could not be started: naturins-linux_1", 1]] }, renderingProperties: { displayColumnNames: null, groupByColumnName: null, type: 1, title: null, description: null } }], status: { message: null, statusId: 0 }, dataProvidersMetadata: null, suggestedUtterances: null } };
```

Comment:
Can you add some space and formatting to this? Also maybe move it below the suite (If you get complaints about "no-use-before-declare", just turn the rule off. It's actually [discouraged](https://palantir.github.io/tslint/rules/no-use-before-declare/))